src/room.js

```diff
@@ -0,0 +1,37 @@
+class Room {
+
+  constructor (roomname) {
+    this.roomname = roomname;
+    this.endpoints = {};
+  }
+  addEndpoint (endpointId) {
+    this.endpoints[endpointId] = {
+      name: '',
+      permissions: 'CHAT',
+      commsid: ''
+    };
+  }
+  updateEndpointName (endpointId, Name) {
+    this.endpoints[endpointId].name = Name;
+  }
+  updateEndpointPermissions (endpointId, permissions) {
+    this.endpoints[endpointId].permissions = permissions;
+  }
+  updateEndpointCommsID (endpointId, commsid) {
```

Comment:
I know this sounds really pedantic :sweat_smile:, but please can we be consistent with capitalising: `commsID` vs `commsId` vs `commsid`.

Everywhere else, we have capital `I`, lowercase `d` :+1: